source/beamline/AMPVControl.cpp

```diff
@@ -209,6 +209,13 @@ AMControl::FailureExplanation AMPVControl::move(double setpoint) {
 	return NoFailure;
 }
 
+// Start a move to the value setpoint:
+AMControl::FailureExplanation AMPVControl::move(const QString &stringSetPoint)
```

Comment:
@dretrex in the old Amptek codes, it added a function `setString()` to the AMPVControl, so that the Detector manager can write string to some PVs.
```
void AMPVControl::setString(const QString &string){
	writePV_->setValue(string);
}
```

In stead of name the function `setString`, I overloaded the function `move()` with the ability to write a string